play/src/front/Components/Input/Input.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+    import InfoButton from "./InfoButton.svelte";
+
+    export let id: string | undefined = undefined;
+    export let label: string;
+    export let placeholder = "";
+    export let onChange = () => {};
+    export let disabled = false;
+    export let type: "text" | "url" = "text";
+    export let value: string | null | undefined;
+    export let onClick = () => {};
+    export let variant: "light" | "" = "";
+    export let taille: "xs" | "sm" | "lg" | "" = "";
+    export let side: "left" | "" = "";
+
+    const SLOTS = $$slots;
+    console.log(SLOTS);
+
+    let uniqueId = id || `input-${Math.random().toString(36).substr(2, 9)} `;
+</script>
+
+<div class="flex flex-col">
+    <div class="input-label">
+        <label for={uniqueId} class="grow">{label}</label>
+
+        {#if SLOTS.info}
+            <InfoButton>
+                <slot name="info" />
+            </InfoButton>
+        {/if}
+    </div>
+
+    {#if type === "text"}
+        <div class=" relative flex grow">
+            <input
+                id={uniqueId}
+                type="text"
+                class="grow input-text input-icon "
+                class:input-icon-left={side === "left"}
+                class:input-switch-light={variant === "light"}
+                class:input-text-xs={taille === "xs"}
+                class:input-text-sm={taille === "sm"}
+                class:input-text-lg={taille === "lg"}
+                bind:value
+                {placeholder}
+                on:change={onChange}
+                on:click={onClick}
+                {disabled}
+            />
+            {#if SLOTS.right}
+                <slot name="right" />
+            {/if}
```

Comment:
Rename the "right" slot like this: "input-append".

Also, add a "input-prepend" slot to add an icon at the beginning of the input field